fmt/testdata/WhileStatement/fmt.sol

```diff
@@ -24,11 +24,15 @@ contract WhileStatement {
 
         uint256 someLongVariableName;
         while (
-            someLongVariableName < 10 && someLongVariableName < 11 &&
-            someLongVariableName < 12
+            someLongVariableName
```

Comment:
I don't really like this. I think we would need some better analysis where we wanna put breakpoints...